lightning-tag/user-consent.md

```diff
@@ -0,0 +1,60 @@
+# User Consent Guide
+
+## Passing Consent Signals Into Lotame
+
+For clients that utilize Lotame's Consent Management Product, consent signals can be passed directly into Lightning Tag. Use this feature to inform Lotame of each user's consent preference and how they are handled within the DMP.
+
+```javascript
+// Callback that runs when setConsent() completes that
+// allows webpage-specific processing if needed at this event
+function consentCb(returnData) {};
+
+// Set these as true or false as per the signals from your User Consent process
+var customerConsents = {
+  analytics: true,
+  crossdevice: true,
+  datasharing: true,
+  targeting: true
+};
+window.lotame_<lotameClientId>.setConsent(consentCb, <lotameClientId>, customerConsents);
```

Comment:
the lotameClientId second parameter is not necessarily the same id, should most likely reference that as consent client id and indicate they should use the client id that manages consent for the client. This could be a child client directly or a parent client.